### ISSUE TITLE
Sticky ads with ad-sense are having 11 instances 

### ISSUE THREAD

**Issue by studboo** [ahmedkaludi/ads-for-wp#175]
Hi,
when using sticky ads, they have 11 instances, I have to click 11 times to close the ads.
Link www.studboo.com/amp (look for the sticky ads at the bottom, they have 11 instances).

Screen shot here : http://studboo.com/wp-content/uploads/2019/04/11-instaces.png

**Comment by studboo** [comment]
I was referred here from : https://wordpress.org/support/topic/the-sticky-ads-on-mobile-amp-is-having-11-instances/

I successfully installed plugin and it is working fine, when I am using sticky ads with amp, the sticky ads are having 11 instances, user has to click 11 times to close them.

Website http://www.studboo.com and amp version here : http://www.studboo.com/amp

When using amp, the sticky ads on the bottom have to be closed 11 times, I am unable to figure out what am I doing wrong.

I have reset everything and did everything from scratch, still there are 11 instances. I have tried many devices to check and the result is the same. I am reading on Google about amp and your plugin, still not able to resolve sticky ads issue.

I want sticky ads closed button to be clicked just once.

Screen shot here : http://studboo.com/wp-content/uploads/2019/04/11-instaces.png
